src/components/AppNavigation/CalendarList/CalendarListItemSharingPublishItem.vue

```diff
@@ -61,7 +61,9 @@
 			<ActionInput
 				v-if="showEMailInput"
 				icon="icon-mail"
-				@submit.prevent.stop="sendLinkViaEMail" />
+				@submit.prevent.stop="sendLinkViaEMail">
+				{{ $t('calendar', 'Enter one address') }}
```

Comment:
I tried something more descriptive first, but the place is very limited.

Later on it would be nice to allow users to enter a comma separated list of email addresses (like up to 25 at a time)

Reply:
In that case we may use some kind of popup so we can also add an eventual message?